src/components/Main/Main.tsx

```diff
@@ -116,19 +119,13 @@ function Main() {
       // At this point the scenario params have been captured, and we can clean up the URL.
       updateBrowserURL('/')
     }
-  }, [])
-
-  const [debouncedRun] = useDebouncedCallback(
-    (params: AllParams, scenarioState: State, severity: SeverityTableRow[]) =>
-      runSimulation(params, scenarioState, severity, setResult, setEmpiricalCases),
-    500,
-  )
+  }, [autorunSimulation, debouncedRun, scenarioState, severity]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
The current version `[]` gives run once (componentDidMount) semantics which is what I think we want to limit the LocalStorage and URL futzing.

Reply:
Should I set it to [] then ?